src/components/Products/Filters.jsx

```diff
@@ -60,14 +64,16 @@ const Filters = ({
             Departamentos
           </Link>
           {selectedFilter === 'department' && (
-            <div className="border border-black rounded-md w-full pl-2 max-h-96 overflow-y-scroll">
+            <div className="border border-black rounded-md w-full max-h-96 overflow-y-scroll">
               {departments.map((department) => (
                 <div
                   key={department.id}
                   onClick={(e) => handleDepartment(e, department.en)}
-                  className="py-2 cursor-pointer text-black underline hover:no-underline"
                 >
-                  {department.es}
+                  {department.en === selectedDepartment
+                    ? <SelectedItemFilter item={department.es} />
+                    : <ItemFilter item={department.es} />
+                  }
```

Comment:
If you apply my changes, this condition is avoid and you can do:

```suggestion
                  <ItemFilter item={department.es} selectedDepartment={selectedDepartment} />
                  
```

Reply:
Same in other similar blocks of code

Reply:
ready